.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature Request
+about: Suggest a feature / QOL improvement
+title: "[Feature] - Short feature summary"
+labels: enhancement
+assignees: ''
+
+---
+
+###### Checklist
```

Comment:
It's a trivial matter, but I think `#### Checklist` is better for consistency with bug report template.

Reply:
Whoops, missed that. Fixed.